### ISSUE TITLE
Interpolated colormap

### ISSUE THREAD

**Issue by tdegeus** [schulzch/colorbrewercpp#1]
Great initiative! I am actually looking for an interpolated version of this, with say 256 colors per map. Have you thought about doing this?

Thanks!
Tom

**Comment by schulzch** [comment]
What kind of color scale are you looking for? I see a couple of options:
- Categorical data: don't do it! Distinguishing more than 12-16 colors for categorical data is really tough, even for non-colorblind people.
- Continuous scale: take some interpolation code from [chroma.js](https://github.com/gka/chroma.js/) and use some seed colors.
- There are some nice alternatives to [ColorBrewer](http://colorbrewer2.org/). Espeically the algorithmic ones, like [ColorCat](http://colorcat.org/) and [Colorgorical](http://vrl.cs.brown.edu/color) might fit your needs.

Note that color perception research contains quite a bit of _faith_, so be careful ;-)

**Comment by tdegeus** [comment]
Thank you for your quick response, and sorry for my delayed response. I like your suggestions a lot, it seems that people have put some thought in it. Though, as you point out, it is difficult to judge them by their scientific contributions.

I ended up being very pressed for time, so I did some linear interpolation of ColorBrewer. This could of course still be improved. Here is the code: [cppcolormap](https://github.com/tdegeus/cppcolormap). 

My code could still learn from your Qt integration, which I like lot. However, based on the header-file in this repository I do not have enough details it seems. 

Thanks!
